ackermann_steering_controller/test/test_ackermann_steering_controller.hpp

```diff
@@ -214,36 +202,45 @@ class AckermannSteeringControllerFixture : public ::testing::Test
   void subscribe_and_get_messages(ControllerStateMsg & msg)
   {
     // create a new subscriber
+    ControllerStateMsg::SharedPtr received_msg;
     rclcpp::Node test_subscription_node("test_subscription_node");
-    auto subs_callback = [&](const ControllerStateMsg::SharedPtr) {};
+    auto subs_callback = [&](const ControllerStateMsg::SharedPtr cb_msg) { received_msg = cb_msg; };
     auto subscription = test_subscription_node.create_subscription<ControllerStateMsg>(
       "/test_ackermann_steering_controller/controller_state", 10, subs_callback);
+    rclcpp::executors::SingleThreadedExecutor executor;
+    executor.add_node(test_subscription_node.get_node_base_interface());
+    received_msg.reset();
+    ASSERT_FALSE(received_msg);
```

Comment:
It shouldn't be a problem unless you spin in between. if you spin yes, then it would reset and be a problem. I was just resetting and added assert to be sure nothing much

Reply:
Since it is a local variable... does it matter at all if we not reset it?

Reply:
Nope, it doesn't matter. I can remove it

Reply:
Thanks. Done